### PR TITLE
feat!: VOL-4507 Upgrade to Doctrine 2.19.5

### PR DESCRIPTION
**Pull request by ilindsay**
## Description

Upgrade to Doctrine 2.19.5

Related issue: [VOL-4507](https://dvsa.atlassian.net/browse/VOL-4507)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "beberlei/doctrineextensions": "^1.0",
         "caseyamcl/guzzle_retry_middleware": "^2.7.1",
         "doctrine/doctrine-orm-module": "^5",
-        "doctrine/orm": "~2.11.0",
+        "doctrine/orm": "2.19.5",
```
Comment (JoshuaLicense) [inline composer.json:15]: Any issue with expanding this constraint?

```suggestion
        "doctrine/orm": "^2.19.5",
```

Comment (ilindsay) [reply composer.json:15]: Oops, hadn't meant to remove that. Is `^` correct though...that could eventually give us 2.20, 2.21 etc, while `~` will keep us on 2.19.

Comment (JoshuaLicense) [reply composer.json:15]: Doctrine follows semantic versioning so in "theory" we'd be okay on any 2.x version.

**Review by fibble — APPROVED** [review]

**Review by JoshuaLicense — APPROVED** [review]

**Comment by fibble** [comment]
@ilindsay - looks good but lockfile conflict needs resolving

**Review by JoshuaLicense — APPROVED** [review]

**Pull request merged**